spec/support/tracer_helpers.rb

```diff
@@ -31,20 +31,14 @@ def get_test_writer(options = {})
   end
 
   # Return some test traces
```

Comment:
I had refactored this helper when implementing a different approach for this PR. I think it's a valuable refactor this test method, so I left it here.